src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PUblishView.cs

```diff
@@ -63,6 +65,8 @@ private void PopulateView()
                 {
                     Logger.Trace("GetCurrentUser");
 
+                    isLoading = true;
```

Comment:
I don't have a good reason for using a different flag, mental clarity if nothing else...